### PR TITLE
Fix the recipe to correctly cargo dependencies

### PR DESCRIPTION
**Pull request by mdrodrigo**
From the kirkstone release, the compile task does not have the network enabled.

This commit changes the following:
Cargo dependencies are downloaded in the fetch task; Easysplash revision is updated.

Signed-off-by: Rodrigo M. Duarte <rodrigo.duarte@ossystems.com.br>

### CODE REVIEW

File: recipes-core/easysplash/files/0001-Remove-Cargo.toml-panic-abort-flag.patch [hunk 1]
```diff
@@ -0,0 +1,22 @@
+From 74b42bc52f910607dfc63f11ecf36e65934cbfc7 Mon Sep 17 00:00:00 2001
+From: "Rodrigo M. Duarte" <rodrigo.duarte@ossystems.com.br>
+Date: Tue, 31 Jan 2023 10:40:04 -0300
+Subject: [PATCH] Remove Cargo.toml panic abort flag
```
Comment (otavio) [inline recipes-core/easysplash/files/0001-Remove-Cargo.toml-panic-abort-flag.patch:4]: Send this change to the project itself so we avoid the patch.

File: recipes-core/easysplash/easysplash_2.0.0.bb [hunk 2]
```diff
@@ -6,6 +9,137 @@ require easysplash-common-2.0.inc
...
+    file://0001-Remove-Cargo.toml-panic-abort-flag.patch \
+"
+
+LICENSE = "Apache-2.0|MIT"
+LIC_FILES_CHKSUM = "file://LICENSE-APACHE-2.0;md5=1836efb2eb779966696f473ee8540542"
+LIC_FILES_CHKSUM = "file://LICENSE-MIT;md5=b377b220f43d747efdec40d69fcaa69d"
+
+SUMMARY = "easysplash"
+HOMEPAGE = "ossystems.com.br"
+LICENSE = "CLOSED"
```
Comment (otavio) [inline recipes-core/easysplash/easysplash_2.0.0.bb:141]: Has this been added automatically.

**Review by otavio — CHANGES_REQUESTED** [review]

File: recipes-core/easysplash/files/0001-Remove-Cargo.toml-panic-abort-flag.patch (hunk 1 above)
Comment (mdrodrigo) [reply recipes-core/easysplash/files/0001-Remove-Cargo.toml-panic-abort-flag.patch]: Fixed at source: 
https://github.com/OSSystems/EasySplash
81a44392efe15fd12ea673f1c828a52e95103eea

File: recipes-core/easysplash/easysplash_2.0.0.bb (hunk 2 above)
Comment (mdrodrigo) [reply recipes-core/easysplash/easysplash_2.0.0.bb]: Removed

**Pull request closed without merging**